code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/NotificationVisibility.kt

```diff
@@ -14,9 +14,9 @@ package com.adobe.marketing.mobile.notificationbuilder.internal
 import androidx.core.app.NotificationCompat
 
 enum class NotificationVisibility(val value: Int, val stringValue: String) {
-    VISIBILITY_PRIVATE(NotificationCompat.VISIBILITY_PRIVATE, "VISIBILITY_PRIVATE"),
-    VISIBILITY_PUBLIC(NotificationCompat.VISIBILITY_PUBLIC, "VISIBILITY_PUBLIC"),
-    VISIBILITY_SECRET(NotificationCompat.VISIBILITY_SECRET, "VISIBILITY_SECRET");
+    VISIBILITY_PRIVATE(NotificationCompat.VISIBILITY_PRIVATE, "PRIVATE"),
+    VISIBILITY_PUBLIC(NotificationCompat.VISIBILITY_PUBLIC, "PUBLIC"),
+    VISIBILITY_SECRET(NotificationCompat.VISIBILITY_SECRET, "SECRET");
```

Comment:
these enum values should be as defined currently
https://github.com/adobe/aepsdk-messaging-android/blob/8fbd21ee588eb152510d0527179b61bd959e8566/code/messaging/src/main/java/com/adobe/marketing/mobile/MessagingPushPayload.java#L46